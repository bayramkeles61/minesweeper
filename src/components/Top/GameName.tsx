import styled from "@emotion/styled";

export const GameName = styled.h1`
  font-size: 2em;
`;
