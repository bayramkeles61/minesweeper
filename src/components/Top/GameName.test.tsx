import { it, expect } from "vitest";
import { render } from "@testing-library/react";

import GameName from "./GameName";

it("GameName renders correctly", () => {
  const { asFragment } = render(<GameName>minesweeper</GameName>);

  expect(asFragment()).toMatchSnapshot(`
    <DocumentFragment>
      <h1
        class="text-[2em]"
      >
        minesweeper
      </h1>
    </DocumentFragment>
  `);
});
