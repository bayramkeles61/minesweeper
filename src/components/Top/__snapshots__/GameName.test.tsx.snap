// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GameName renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="css-18kvaxm"
  >
    minesweeper
  </h1>
</DocumentFragment>
`;
